ci/jobs/build_clickhouse.py

```diff
@@ -155,7 +161,9 @@ def main():
         res = results[-1].is_ok()
 
     if res and JobStages.BUILD in stages:
-        Shell.check("sccache --show-stats")
+        run_shell("Env vars", "env | sort")
```

Comment:
I would love to have env vars in the logs, but, it may have some sensitive data (it does not now however)